TappNetwork/nova-dependency-container
# Add array

# Details

Allows nova-dependency-container to work with BooleanGroups as well as allowing arrays to be used for dependencies.

# Prerequisites
1. You have the latest from the CSC branch PG-1607_Hide_Business_Unit
1. `composer install`
1. `npm run dev`

# Steps to test changes
1. Log in to CSC
1. Navigate to admin/resources/users/new
1. Select one of the following roles - `Admin`, `CSC`, `CSC-Admin`
1. Note that the business unit dropdown shows up.

# Issues Addressed

[PG-1607](https://gfm-it.atlassian.net/browse/PG-1607) - Adds the ability to show and hide based on arrays / boolean groups.
